scripts/components/BucketGroups.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import BucketTabs from "./BucketTabs";
+
+
+class DataList extends Component {
```

Comment:
Note that as we don't deal with component state here, we could totally use a stateless component. We're only forced to use classes for *default* exports if we want to have hot reloading.